main.go

```diff
@@ -213,21 +189,12 @@ func main() {
 		FlagdProxyConfig: kph.Config(),
 		Env:              env,
 		FlagdInjector: &flagdinjector.FlagdContainerInjector{
-			Client:           mgr.GetClient(),
-			Logger:           ctrl.Log.WithName("flagd-container injector"),
-			FlagdProxyConfig: kph.Config(),
-			FlagDResourceRequirements: corev1.ResourceRequirements{
-				Limits: map[corev1.ResourceName]resource.Quantity{
-					corev1.ResourceCPU:    cpuLimitResource,
-					corev1.ResourceMemory: ramLimitResource,
-				},
-				Requests: map[corev1.ResourceName]resource.Quantity{
-					corev1.ResourceCPU:    cpuRequestResource,
-					corev1.ResourceMemory: ramRequestResource,
-				},
-			},
-			Image: env.SidecarImage,
-			Tag:   env.SidecarTag,
+			Client:                    mgr.GetClient(),
+			Logger:                    ctrl.Log.WithName("flagd-container injector"),
+			FlagdProxyConfig:          kph.Config(),
+			FlagDResourceRequirements: *resources,
```

Comment:
```suggestion
			FlagdResourceRequirements: *resources,
```

Reply:
It doesn't look like this is used in any of the changed files. Please update all the references to this.